allennlp/service/config_explorer.html

```diff
@@ -0,0 +1,608 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>AllenNLP Configuration Wizard (alpha)</title>
+    <style>
+        div {
+            display: table;
+        }
+
+        * {
+            font-family: sans-serif;
+        }
+
+        h1,
+        h2 {
+            -webkit-font-smoothing: antialiased;
+            -moz-osx-font-smoothing: grayscale;
+            font-weight: 300
+        }
+
+        h2 {
+            font-size: 2em;
+        }
+
+        .optional {
+            color: gray;
+        }
+
+        .required {
+            color: black;
+        }
+
+        span.name {
+            font-weight: bold;
+            margin: 5px;
+        }
+
+        .required.incomplete > span.name {
+            background-color: lightcoral;
+        }
+
+        .annotation {
+            font-size: 90%;
+            margin-left: 10px;
+            margin-right: 10px;
+            color: #2085bc;
+        }
+
+        .prefix {
+            font-size: 75%;
+            margin-left: 10px;
+        }
+
+
+        .default-value {
+            color: #979a9d;
+            font-size: 90%;
+        }
+
+        div#rendered-json {
+            margin: 10px;
+            border: 1px solid black;
+            font-family: monospace;
+            white-space: pre;
+        }
+
+        .config, .list, .dict {
+            margin-top: 5px;
+            margin-left: 40px;
+        }
+
+        .choices-dropdown {
+            margin-left: 20px;
+        }
+
+        .config-item {
+            margin-top: 2px;
+        }
+
+        .dict-item {
+            border: 1px dotted gray;
+        }
+
+        button.subconfigure {
+            margin: 2px;
+        }
+
+        .tippy-content {
+            color: white;
+        }
+    </style>
+  </head>
+  <body>
+    <h2>AllenNLP Configuration Wizard (alpha)</h2>
+
+    <div id="app"></div>
+
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/immutable/3.8.2/immutable.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.26.0/babel.js"></script>
+    <script src="https://unpkg.com/tippy.js@2.5.2/dist/tippy.all.min.js"></script>
+    <script type="text/babel">
+/*
+ConfigItem = {
```

Comment:
Looks like you've replaced these below, right?

Reply:
sorry, that's not javascript code, that's a comment about the types that come from the API, I'll add a comment to that effect